### ISSUE TITLE
IRC/Discord to exchange scripts?

### ISSUE THREAD

**Issue by isherzad** [Masuzu/ZooeyBot#1199]
Hello, I'm curious if there is an IRC or a Discord Server where we could exchange info or scripts or general discussion and questions? Since it it is fairly easy to keep a separate identity from the one you use Granblue with.

**Comment by Masuzu** [comment]
Hello,
It may be time for this. Just to set clear expectations beforehand, do not expect a lot of support personally from me as I would probably be busy somewhere else most of the time and miss messages.

**Comment by zooeyjapan** [comment]
+1 to this :-)

**Comment by talran** [comment]
I'd be down for that, less hassle than remembering to update/push git

**Comment by isherzad** [comment]
@Masuzu Yes, of course. I was thinking more along the lines of script exchange and general help since theres is probably a lot of people keen on this type of subject.

**Comment by ramblinguy** [comment]
I would be open to this as well, as most of the scripting knowledge is pulled from the example scripts. I would love to see what others have written

**Comment by Masuzu** [comment]
Let's give this a try.
AUA as for "Ask Us Anything" https://discord.gg/8XDutQ

**Comment by ramblinguy** [comment]
That link isn't working for me, is it working for anyone else?

**Comment by talran** [comment]
Looks like it was an instant invite, Just need a fresh/perm one

**Comment by filipi159951** [comment]
Is there a new invite? I’d like to join on it

**Comment by Masuzu** [comment]
Oh sorry. Here is an updated link https://discord.gg/7qzeRhE
